### ISSUE TITLE
wrong info on W1PD96185

### ISSUE THREAD

**Issue by eroux** [buda-base/library-issues#279]
W1PD96185 is mistakenly identified as being a dPal spungs-redaction of the Rin chen gter mdzod chen mo, but seems to a reprint from the 1909-1912-mTshur phu edition consisting of 63 volumes. The earlier dPal dpungs prints that served as basis for reproduction consisted of 60/61 vols.; see [Schwieger 1990](https://sakyaresearch.org/sources/3386), introduction

**Comment by karmagongde** [comment]
The images of W1PD96185 are the same as W20578 "a reproduction of the stod-lun mtshur-phu redaction of 'jam-mgon kon-sprul's great work on the unity of the great gter-ma traditions of Tibet. With supplemental texts from the dpal-spuns redaction and other manuscripts." 

Volume no. 63 in W1PD96185 is the same as the volume no. 100 in W20578. Since every text does not have the source printery name in the texts it is very difficult to determine which volumes are stod-lun mtshur-phu redaction and which are dpal-spuns redaction. So I added stod-lun mtshur-phu redaction in W1PD96185. Now both the redaction are there just like in W20578.

**Comment by eroux** [comment]
Thanks!

**Comment by dustcats** [comment]
@karmagongde Yes, apart from the margins the prints are same. W20578 and W1PD96185 are both redactions of the _Rin chen gter mdzod chen mo_ from the mTshur phu edition and W20578 (Paro: Ngodrup and Sherab Drimay, 1976-1980) contains supplements from the Dpal-spungs redaction and other manuscripts which blows up the collection up to 111 volumes.

BDRC W8LS31064 holds a redaction from the dPal spungs prints in 61 volumes and can be taken into comparison to determine that W1PD96185 is also based on the mTshur phu edition.
